### PR TITLE
Documentation: Previewing on an Android device

### PR DESCRIPTION
**Pull request by jeremywiebe**
This PR extends the note about previewing with SSL. It also re-arranges the `README` docs in light of the recent folder moves (native/web).

 **JIRA**: N/A
 **Linked PRs**: N/A

## Changes
- Adds documentation outlining how to preview on an Android device
- Creates [README.md](/README.md)
- Moves some content from [web/README.md](/web/README.md) to [README.md](/README.md)

## How to test-drive this PR
- Just read the doc updates
- Make sure the portions moved to [README.md](/README.md) make sense to be there

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Platform Scaffold
+
+Welcome to the Platform Scaffold. This repo is (currently) organized as follows:
```
Comment (jvoll) [inline README.md]: do we need to say "(currently)" here? I don't think we expect this to change do we?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+| ------------------ | ------------------------------------------------------------------- |
+| [/web](/web)       | Contains the web (Mobify Progressive Web SDK) scaffold              |
+| [/native](/native) | Contains the native (Mobify Progressive App SDK, or Astro) scaffold |
+
+## Requirements
+
+- [Git](https://git-scm.com/)
+- We recommend you use [nvm](https://github.com/creationix/nvm#installation) to
+manage node and npm versions.
+- node 6.x LTS
```
Comment (jvoll) [inline README.md]: Is astro on node 6?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+The development server uses a self-signed SSL certificate which is
+valid, but treated as suspect by browsers. This means that we must
+create and reconfirm security exceptions for it, and avoid localhost
+for certain use cases (such as service workers).
+
+To add the certificate to the Mac system trust store and make the
+browsers accept it, do the following:
+
+1. In the root of the project directory, run `open node_modules/webpack-dev-server/ssl/server.crt`.
```
Comment (jvoll) [inline README.md]: there won't be a `node_modules` directory at your project root anymore right?

File: README.md (hunk 1 above)
Comment (jeremywiebe) [inline README.md]: Agreed. I wrote this thinking about other parts of the platform but then added the table which will be easy to extend. Removed "(currently)"

File: README.md (hunk 2 above)
Comment (jeremywiebe) [inline README.md]: The branch we're using in this repo does... and will across the board soon.

File: README.md (hunk 3 above)
Comment (jeremywiebe) [inline README.md]: Good catch!

**Pull request merged**